packages/blockchain-utils-linking/src/tezos.ts

```diff
@@ -0,0 +1,117 @@
+import { AccountID } from 'caip'
+import { AuthProvider } from './auth-provider'
+import { getConsentMessage, LinkProof } from './util'
+import { hash } from '@stablelib/sha256'
+import * as uint8arrays from 'uint8arrays'
+import { SetProviderOptions, TezosToolkit } from '@taquito/taquito'
+import { char2Bytes } from '@taquito/utils'
```

Comment:
Please use `uint8arrays` to convert utf-8 to a bytestring.

Reply:
Cant't. `char2Bytes` does a custom `string` to `string` conversion.

Reply:
It's really simple: https://github.com/ecadlabs/taquito/blob/master/packages/taquito-utils/src/taquito-utils.ts#L249
Please use `uint8arrays`!

Reply:
`uint8arrays.toString(uint8arrays.fromString(input, 'utf8'), 'base16')` would work @Geo25rey

Reply:
I want to confirm it produces the same output then I will fix. I will get to it soon.

Reply:
Works the same!